### ISSUE TITLE
[2019.1.7f1] InitializeIfNeeded: Object reference not set to an instance of an object

### ISSUE THREAD

**Issue by jcrombez** [forestrf/UnityAnimatorEvents#1]
Hi ! I tried using your asset, but got the following error when i expand the `AnimatorEvent` component.

```
NullReferenceException: Object reference not set to an instance of an object
AnimatorEventEditor.InitializeIfNeeded () (at Assets/Plugins/UnityAnimatorEvents-master/StateMachineBehaviours/Editor/AnimatorEventEditor.cs:13)
AnimatorEventEditor.OnInspectorGUI () (at Assets/Plugins/UnityAnimatorEvents-master/StateMachineBehaviours/Editor/AnimatorEventEditor.cs:35)
UnityEditor.UIElements.InspectorElement+<CreateIMGUIInspectorFromEditor>c__AnonStorey1.<>m__0 () (at C:/buildslave/unity/build/Editor/Mono/Inspector/InspectorElement.cs:462)
UnityEngine.GUIUtility:ProcessEvent(Int32, IntPtr)
```

**Comment by forestrf** [comment]
I think I found the problem. The array accessed in that line isn't initialized by the code because I expected Unity to initialize it. It doesn't because I'm using a custom inspector, so it ends up trying to get the length of an array that is null. It should be fixed in the commit b1fc792, can you see if it did the trick?

**Comment by jcrombez** [comment]
Yes it fixes the error :)
